libs/sdk-ui-dashboard/src/plugins/plugin.ts

```diff
@@ -0,0 +1,221 @@
+// (C) 2021 GoodData Corporation
+
+import { IDashboardCustomizationProps, IDashboardEventing, IDashboardThemingProps } from "../presentation";
+import { IAnalyticalBackend } from "@gooddata/sdk-backend-spi";
+import { ObjRef } from "@gooddata/sdk-model";
+import { DashboardEventType, IDashboardEvent } from "../model";
+import React from "react";
+
+/**
+ * @alpha
+ */
+export interface IDashboardPluginMetadata {
+    /**
+     * Version of the SPI that is realized by the plugin.
+     */
+    readonly _spiVersion: "1.0";
+
+    /**
+     * Specify human-readable name of the plugin.
+     */
+    readonly displayName: string;
+
+    /**
+     * Optionally specify human-readable short description of the plugin. This is typically a one- or two-line description
+     * of the plugin, what it brings, what it does.
+     */
+    readonly shortDescription?: string;
+
+    /**
+     * Human-readable long description of the plugin.
+     */
+    readonly longDescription?: string;
+
+    /**
+     * Developer-assigned name of the plugin that will be used
+     *
+     * If not specified the debug name falls back to display name.
+     */
+    readonly debugName?: string;
+
+    /**
+     * Author of the plugin. This should ideally contain name and contact (email) for the author.
+     */
+    readonly author?: string;
```

Comment:
Should we make this more structured? So that we can more easily get emails of plugin authors when needed for some comms?